js/source/worker.js

```diff
@@ -14,7 +14,7 @@ var rewind = require('geojson-rewind');
 var GeoJSONWrapper = require('./geojson_wrapper');
 var vtpbf = require('vt-pbf');
 
-module.exports = function(self) {
+module.exports = function MapboxGlWorker(self) {
```

Comment:
If I understand https://github.com/borisirota/webworkify-webpack/issues/9 correctly, this shouldn't be necessary with webworkify-webpack 1.1.1, correct?